documentation/book/security.adoc

```diff
@@ -1,27 +1,37 @@
 This makes it possible to encrypt data transferred between brokers (interbroker communication) and between clients and brokers.
+For the internal components, TLS certificates are also used for authentication.
 
 The Cluster Operator sets up the SSL/TLS certificates to provide this encryption and authentication.
 
 === Certificates
 
-Each Kafka broker needs its own private and public keys in order to support encryption.
+Each component needs its own private and public keys in order to support encryption.
 The public key has to be signed by a certificate authority (CA) in order to have a related X.509 certificate for providing server authentication and encrypting the communication channel with the client (which could be another broker as well).
-More precisely, each broker has two different certificates (and related private keys), one for interbroker communication  and one for communication with clients.
-These broker certificates are signed using different CAs: The "internal-ca" signs the certificates used for interbroker communication, and the "clients-ca" signs the certificates used for client communication.
+All components are signed by a Certification Authority (CA) called _cluster CA_.
+Another CA is used for authentication of Kafka clients connecting to Kafka brokers.
+This CA is called _clients CA_.
 The CAs themselves use self-signed certificates.
-The "internal-ca" is used across multiple clusters, that is, there is a single "internal-ca" for each instance of a cluster operator. 
-The "clients-ca" is specific to a particular Kafka cluster.
 
 All the generated certificates are saved as Secrets in the {ProductPlatformName} cluster, named as follows:
 
-* `internal-ca`: contains the private and public keys, so the self-signed certificate, used for signing broker certificates used for interbroker communication. It is common to all the Kafka clusters deployed by the Cluster Operator.
-* `<cluster-name>-kafka-clients-ca`: contains the private and public keys, so the self-signed certificate, used for signing broker certificates used for communicating with clients. It is specific for each deployed Kafka cluster as specified in the <cluster-name> prefix.
-* `<cluster-name>-kafka-cert`: contains only the public keys, so the self-signed certificate, used for signing broker certificates used for communicating with clients. This Secret should be used by the final Kafka cluster user in order to extract this certificate to put into the the client's truststores for verifying broker identities (server authentication).
-* `<cluster-name>-kafka-brokers-internal`: contains all the brokers private and public keys (certificates signed with "internal-ca") used for interbroker communication.
-* `<cluster-name>-kafka-brokers-clients`: contains all the brokers private and public keys (certificates signed with specific cluster "clients-ca") used for communicating with clients.
+`<cluster-name>-cluster-ca`::
```

Comment:
In this list we say what the keys are named only for a couple of the secrets. Should we include this info for all secrets and all keys?

Reply:
I removed all the names. In the places they have some significance we should add more details (such as extracting the CA public key).